.ci/deps.shellcheck.sh

```diff
@@ -24,3 +24,6 @@ else
     echo "Using cached ShellCheck $EXISTING_VERSION"
   fi
 fi
+
+cabal update && cabal install cabal-install-1.22.9.0
+cabal install ghc-mod
```

Comment:
oh I think this should go to `deps.sh` instead of this spellcheck install script^^